app/bundles/CoreBundle/Model/IteratorExportDataModel.php

```diff
@@ -68,8 +60,8 @@ public function next()
         if ($this->position === $this->totalResult) {
             $data              = new DataExporterHelper();
             $this->data        = $data->getDataForExport($this->total, $this->model, $this->args, $this->callback);
-            $this->total       = $this->total + count($this->data);
-            $this->totalResult = count($this->data);
+            $this->totalResult = $this->data ? count($this->data) : 0;
+            $this->total       = $this->total + $this->totalResult;
             $this->position    = 0;
             ++$this->page;
```

Comment:
Could you please remove this line. Variable is not used and it's producing
```
PHP Notice - Undefined property: Mautic\CoreBundle\Model\IteratorExportDataModel::$page" at /Users/lukas.drahy/dev/community-fork/app/bundles/CoreBundle/Model/IteratorExportDataModel.php line 66
```
for me now.